### PR TITLE
Update Gruntfile to allow for phrase pull and push #172927249

### PR DESCRIPTION
**Pull request by akegan**
This PR:
1. Removes use of tmp files in `grunt translations`
1. Adds `phrasePush` and `phrasePull` commands to Gruntfile.js to automatically push and pull from Phrase with only one command. 
1. Switches label.other to label._other because otherwise Phrase assumes that it's the "other" option for pluralization of "label"

Still TODO:

- [ ] Verify that we're procuring Phrase
- [x] Add instructions to the readme 

Things we could do in the future (We can do them in [this ticket](https://www.pivotaltracker.com/story/show/173635532)):
1. Figure out a [branching scheme](https://help.phrase.com/help/branches)
1. Establish best practices around tags/labels for uploads 
1. Setup automated Github sync

# Review instructions
**Verify that grunt translations still works**
1. Try removing usage of a key throughout the code, run `grunt translations`, and verify that it's deleted from all of the locale files
1. Try messing up the sorting of one of the locale files, run `grunt translations`, and verify that it re-sorts

**Test out the README instructions and phrase push/pull functionality**
1. Follow instructions in the readme to get setup
1. Try running `grunt phrasePull` and verify that there's no diff
1. Try updating an english string in phrase, and then run phrase Pull and verify that it's pulled down OK
1. Try making an update in a locale-en string locally, try running phrase push and verify that it pushes up OK. (maybe fix the string you just messed up)
1. Verify that you didn't end up with any net changes on phrase or locally. 
1.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -89,6 +89,18 @@ In order to test caching locally,
 
 To run stress testing against the Salesforce instance, refer to the documentation in the [stress testing folder](load_testing/load_testing.md)
 
+## Pulling and pushing translations from [our project on Phrase](https://app.phrase.com/accounts/city-county-of-san-francisco/projects/dahlia-sf-dahlia-web/dashboard)
+To get started working with our Phrase translations, you will need to:
+
+1. Install the Phrase CLI with `brew tap phrase/brewed && brew install phrase`
+1. [Create and save a new access token](https://app.phrase.com/settings/oauth_access_tokens)
```
Comment (james-wills-sf) [inline README.md:96]: Can you explain where to store this token? I haven't created a Phrase account yet so I haven't actually read the info at that link

Comment (akegan) [reply README.md:96]: In lastpass 😕

Comment (james-wills-sf) [reply README.md:96]: Oh I missed the piece just under this that explains you need to pass `--phraseAccessToken=[your access token]` with each command. Okay!

Comment (akegan) [reply README.md:96]: I'll update to make it clearer

Comment (akegan) [reply README.md]: updated

Comment (james-wills-sf) [reply README.md]: Thanks, looks good

**Comment by akegan** [comment]
going to merge this in because it has improvements even if we don't end up using phrase. and I'm hoping that we will use it!

**Pull request merged**